README.md

```diff
@@ -14,6 +14,20 @@ We believe engineers should reason more scientifically about code performance, a
 More detailed documentation can be found [here.](https://workday.github.io/warp-core)
 
 ## Getting Started
+
+### Important and Subtle:
+In order for `-PallScalaVersions` to work as expected, you must clone this into a non-default directory, ie a directory that is not equal to the repository name.
```

Comment:
alright this should be good to go, this stuff with the clone directory and subtle 2 gradle issues took some time to figure out

Reply:
@sarcasticsimba @rooeque @tsoppet  just making sure y'all are aware of this

Reply:
oh wow okay that's good to know